core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -839,26 +839,27 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
     assume(!(Utils.isWindows))
     withTempDir { dir =>
       val discoveryScript = createTempScriptWithExpectedOutput(dir, "resourceDiscoveryScript",
-        """{"name": "gpu","addresses":["0", "1", "2"]}""")
+        """{"name": "gpu","addresses":["0", "1", "2", "3", "4", "5", "6", "7", "8"]}""")
```

Comment:
why was this changed?

Reply:
Previously, this discoveryScript was used for those 3 executors directly. And now, when we get resources scheduling in Standalone, this discoveryScript should be used by Worker, otherwise executor can't be launched due to no confined resources for worker.  As a result, we need to prepare 9 isolated resource addresses for 3 workers in the same host  in order to launch 3 executor.